.circleci/config.yml

```diff
@@ -4,12 +4,16 @@ orbs:
   slack: circleci/slack@3.4.2
 
 references:
-  shared-environment: &shared-environment
-    CIRCLE_ARTIFACTS: /tmp/artifacts
-    CIRCLE_TEST_REPORTS: /tmp/test_results
-
-  docker-image: &docker-image
-    - image: circleci/node:12.16.2-browsers
+  defaults: &defaults
```

Comment:
I "inlined" `shared-environment` and `docker-image` because nobody is reusing them, so there is no point in having them a separate yaml entry.